tests/sles4sap/publiccloud/add_server_to_hosts.pm

```diff
@@ -0,0 +1,41 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Deployment steps for qe-sap-deployment
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use qesapdeployment;
+
+sub run {
+    my ($self, $run_args) = @_;
+    my $instance = $run_args->{my_instance};
```

Comment:
This will make this module usable only on test scenarios where we have 1 SUT in the cloud, and not when we have clusters there. The intention is to extend the module to more instances in the cloud in upcoming PRs or use a different module for scenarios such as HanaSR?